### PR TITLE
Include docs on building IB/RDMA support

### PR DESCRIPTION
**Pull request by quasiben**
When https://github.com/rapidsai/ucx-split-feedstock/pull/40 is merged we should have docs on how to build UCX properly for IB/RDMA support

### CODE REVIEW

File: docs/source/install.rst
Comment (jakirkham) [inline docs/source/install.rst:77]: Could just do `curl -LO` to download them to a file of the same name in the current working directory.

```suggestion
    curl -LO https://raw.githubusercontent.com/rapidsai/ucx-split-feedstock/master/recipe/add-page-alignment.patch
    curl -LO https://raw.githubusercontent.com/rapidsai/ucx-split-feedstock/master/recipe/ib_registration_cache.patch
```

File: docs/source/install.rst
Comment (jakirkham) [inline docs/source/install.rst:78]: Could just do `curl -LO` to download them to a file of the same name in the current working directory.

```suggestion
    curl -LO https://raw.githubusercontent.com/rapidsai/ucx-split-feedstock/master/recipe/add-page-alignment.patch
    curl -LO https://raw.githubusercontent.com/rapidsai/ucx-split-feedstock/master/recipe/ib_registration_cache.patch
    git apply ib_registration_cache.patch && git apply add-page-alignment.patch
```

File: docs/source/install.rst
Comment (jakirkham) [inline docs/source/install.rst:133]: Is this mean to still be here or can this be dropped?

```suggestion
```

**Comment by jakirkham** [comment]
Thanks Ben! 😄 Made a couple minor suggestions above.

File: docs/source/install.rst
Comment (pentschev) [inline docs/source/install.rst:101]:
```suggestion
To build UCX with IB/RDMA support, include the ``--with-rdmacm`` and ``--with-verbs`` build flags.  For example:
```

Comment (pentschev) [reply docs/source/install.rst:101]: `cm` is deprecated, according to a comment by Yossi in Slack:

> there are several things called "cm" in different contexts, in particular usage of libibcm package and --with-cm config param are considered deprecated and will probably be removed

File: docs/source/install.rst
Comment (pentschev) [inline docs/source/install.rst:108]:
```suggestion
```

File: docs/source/install.rst
Comment (pentschev) [inline docs/source/install.rst:42]:
```suggestion
    As of version 0.15, the UCX conda package build will no longer include IB/RDMA support.  This is largely due in part to compatibility issues
```

File: docs/source/install.rst
Comment (pentschev) [inline docs/source/install.rst:75]:
```suggestion
    # apply UCX IB registration cache patches, improves overall
    # IB performance when using a memory pool
```

**Review by pentschev — CHANGES_REQUESTED** [review]
Thanks @quasiben , I've added some changes, but it looks good overall. :)

**Comment by pentschev** [comment]
@quasiben I pushed https://github.com/rapidsai/ucx-py/pull/584/commits/84c0db8e75263f9db60a96630e5320cfdb876cf6 too, I noticed you accepted the other suggestion to get rid of `--with-cm` so I assumed you only missed this. Hope you don't mind. :)

**Comment by quasiben** [comment]
> @quasiben I pushed 84c0db8 too, I noticed you accepted the other suggestion to get rid of --with-cm so I assumed you only missed this. Hope you don't mind. :)

Not at all

**Review by pentschev — APPROVED** [review]
LGTM, thanks @quasiben .

**Comment by quasiben** [comment]
Thanks @pentschev and @jakirkham for the review.  I pushed one more commit to update text a bit and noticed a small error.  If you have time can you review once more and merge if you agree?

**Comment by pentschev** [comment]
> Thanks @pentschev and @jakirkham for the review. I pushed one more commit to update text a bit and noticed a small error. If you have time can you review once more and merge if you agree?

LGTM, I'm fine merging as is, will leave it up to @jakirkham once he gets a chance to take a final look.

**Comment by quasiben** [comment]
I'm fine waiting

**Comment by jakirkham** [comment]
Thanks Ben! Also thanks Peter for reviewing! 😄

**Pull request merged**
